tests/spec/w3c/headers-spec.js

```diff
@@ -11,20 +11,82 @@ import {
 const findContent = string => {
   return ({ textContent }) => textContent.trim() === string;
 };
+
+/**
+ * @param {Node} node
+ */
+function collapsedTextContent({ textContent }) {
+  return textContent.replace(/\s+/g, " ");
+}
+
+function contains(el, query, string) {
+  return [...el.querySelectorAll(query)].filter(child =>
+    collapsedTextContent(child).includes(string)
+  );
+}
+describe("W3C - Headers for 2021 Process", () => {
+  afterAll(flushIframes);
+
+  it("has a details and summary", async () => {
+    const opts = makeStandardOps({ specStatus: "FPWD" });
+    const doc = await makeRSDoc(opts);
+    const details = doc.querySelector(".head details");
+    const summary = doc.querySelector(".head summary");
+    expect(details).toBeTruthy();
+    expect(summary).toBeTruthy();
+  });
+
+  it("links to the 'kinds of documents' only for W3C documents", async () => {
+    for (const specStatus of [
+      "FPWD",
+      "LCWD",
+      "WD",
+      "CR",
+      "CRD",
+      "PR",
+      "REC",
+      "NOTE",
+    ]) {
```

Comment:
```suggestion
    const statuses = ["FPWD", "LCWD", "WD", "CR", "CRD", "PR", "REC", "NOTE"];
    for (const specStatus of statuses) {
```

Reply:
Prettier did it.

Reply:
I think we will have the same problem when we add more statuses. I might import them instead.